### ISSUE TITLE
Action "Synchronize calendars" from contex menu doesn't work

### ISSUE THREAD

**Issue by isorochan** [jobisoft/TbSync#181]
environment:
TbSync version: 0.7.19
Thunderbird version:60.3.1 (64-bit) 

## Expected behavior
After clicking on the item "Synchronize calendars" should be started the synchronization process
## Actual behavior
After clicking on "Synchronize calendars" nothing happens 

## Steps to reproduce
1. Invoke context menu in the calendar area 
2. Click on "Synchronize calendars"

**Comment by jobisoft** [comment]
Yep, does not work. For now you have to open the TbSync Manager to manually sync a calendar. 

Will be fixed, but not know (requires to use a custom calendar class and not just using the default one, will be done together with all the meeting bugs)

**Comment by jasmines** [comment]
Same here. 
When you fix it, remember to add a sync on start feature/option and an auto-sync every x minutes...!

**Comment by jobisoft** [comment]
Auto sync is there already:
![image](https://user-images.githubusercontent.com/5830621/48948543-1b973680-ef35-11e8-95be-4b356f7c27f4.png)

Set that field to any value but 0. For example "5" to sync every 5 minutes.

**Comment by bmarinov** [comment]
I just got bitten by this, didnt realize it is not working and kept spamming it and going back and forth between TB and outlook.com while trying to figure out what's wrong. :( 

If implementing it is not on the table right now, then maybe a big warning letting users know about this problem would be a good idea? When does the calender even get synchronized if AutoSync interval is set to 0? That was the default for me, sounds like a good way to miss some meetings.

**Comment by mrwensveen** [comment]
Is there a reason not to set the Sync interval to some other value than 0?

I am hesitant to turn autosync on because there might be a good reason it is off by default. If there is no valid reason, could the default be set to 5 minutes, for example? I think there is an expectation that calendar data is synchronized (because otherwise, what's the point?)
